### DISCUSSION TITLE
Kudos

### DISCUSSION THREAD

**Discussion by Shredder5262** [vasqued2/ha-teamtracker-card#81] [General]
Didn't know a more appropriate place to directly post this, but I was just in the process of rebuilding my home assistant from scratch after 2 years of use and i just wanted to take the time to say that this card and team tracker and integration has been f*ckin great. There is SOOOOO much garbage a** integration work put out  on the HACS platform but this espn team tracker card is not one of them. It does what it does and it does it well. Keep up the good work!

**Comment by vasqued2** [comment]
Thanks for the feedback.  I appreciate it.
